types/sanctuary/index.d.ts

```diff
@@ -13,38 +14,40 @@ export as namespace S;
 type Nullable<A> = A | null;
 
 interface Pair<A, B> {
-  '@@type': 'sanctuary/Pair';
+    '@@type': 'sanctuary/Pair';
 }
 
 type Thunk<A> = () => A;
 
-type Fn<A, B>                = (a: A) => B;
-type Fn2<A, B, C>            = (a: A) => (b: B) => C;
-type Fn3<A, B, C, D>         = (a: A) => (b: B) => (c: C) => D;
-type Fn4<A, B, C, D, E>      = (a: A) => (b: B) => (c: C) => (d: D) => E;
-type Fn5<A, B, C, D, E, F>   = (a: A) => (b: B) => (c: C) => (d: D) => (e: E) => F;
-type Fn2_<A, B, C>           = (a: A, b: B) => C;
-type Fn3_<A, B, C, D>        = (a: A, b: B, c: C) => D;
-type Fn4_<A, B, C, D, E>     = (a: A, b: B, c: C, d: D) => E;
-type Fn5_<A, B, C, D, E, F>  = (a: A, b: B, c: C, d: D, e: E) => F;
+type Fn<A, B>               = (a: A) => B;
+type Fn2<A, B, C>           = (a: A) => (b: B) => C;
+type Fn3<A, B, C, D>        = (a: A) => (b: B) => (c: C) => D;
+type Fn4<A, B, C, D, E>     = (a: A) => (b: B) => (c: C) => (d: D) => E;
+type Fn5<A, B, C, D, E, F>  = (a: A) => (b: B) => (c: C) => (d: D) => (e: E) => F;
+type Fn2_<A, B, C>          = (a: A, b: B) => C;
+type Fn3_<A, B, C, D>       = (a: A, b: B, c: C) => D;
+type Fn4_<A, B, C, D, E>    = (a: A, b: B, c: C, d: D) => E;
+type Fn5_<A, B, C, D, E, F> = (a: A, b: B, c: C, d: D, e: E) => F;
```

Comment:
Could you revert these changes and the other formatting changes to make the significant changes easier to review? :)

Reply:
I dropped the commit that used DefinitelyTyped's prettier. Is that better @davidchambers?